config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kinder-discovery.yaml

```diff
@@ -46,7 +46,7 @@ periodics:
           cpu: 2000m
 
 - name: ci-kubernetes-e2e-kubeadm-kinder-discovery-1-28
-  cluster: eks-prow-build-cluster
+  cluster: k8s-infra-prow-build
```

Comment:
these files are autogenerated, from config and temaplates.
adding manually the EKS cluster will not work.

this in particular is left over when we tried to migrate the kubeadm jobs to EKS.

in any, case if we try to do it again we have to modify the config/templates.
push PRs, with the generated diffs in k/kubeadm and k/test-infra.

Reply:
sgtm!

Reply:
/lgtm